test/testapps/BasicTestApp/wwwroot/js/index.js

```diff
@@ -0,0 +1,176 @@
+// We'll store the results from the tests here
```

Comment:
It would be good to rename this file to something like `jsinteroptests.js`

Reply:
Sure